jekyll-assets/scripts/copy-to-clipboard.js

```diff
@@ -48,7 +48,24 @@ window.addEventListener('load', function() {
       if (trigger.parentNode.querySelector('div.line')) {
         var text = extractDoxygenCode(trigger.parentNode);
       } else {
-        var text = trigger.parentNode.querySelector('pre').textContent;
+        var text = trigger.parentNode.querySelector('pre').textContent
+        
+        // if the code snippet represents a console snippet
+        if (trigger.parentNode.querySelector('pre').querySelector('code') != null && trigger.parentNode.querySelector('pre').querySelector('code').getAttribute('data-lang') == 'console') {
```

Comment:
Could you reduce some repetition by doing a single query with a more powerful CSS selector here? e.g.

```javascript
if (trigger.parentNode.querySelector('pre > code[data-lang="console"]')) {
```

Reply:
Thanks, I didn't realize `querySelector` could handle something that complex! That is much cleaner.